src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/ToGerritRunListener.java

```diff
@@ -215,10 +218,10 @@ public synchronized void onStarted(Run r, TaskListener listener) {
                     ((GerritEventLifecycle)cause.getEvent()).fireBuildStarted(r);
                 }
             }
-            if (!cause.isSilentMode()) {
+            GerritTrigger trigger = GerritTrigger.getTrigger(r.getParent());
+            if (!cause.isSilentMode() || (cause.isSilentMode() && !trigger.isSilentComments())) {
```

Comment:
there needs to be a better way to check this than just listing all possibilities, or at least a utility function.